### PR TITLE
Load workflow definitions on conductor startup

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Custom entrypoint added.
Entrypoint statrts conductor, then loads workflow definitions
from workflow directory posting them to conductor API using curl.
All *.json file from workflows directory will be uploaded to conductor.

@bboozzoo @maciejmrowiec @mchalski

### CODE REVIEW

File: conductor/server/entrypoint/start_conductor.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (bboozzoo) [inline conductor/server/entrypoint/start_conductor.sh]: `#!/bin/bash` -e or `set -e`

File: conductor/server/entrypoint/start_conductor.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+/app/startup.sh &
+sleep 60
+
+# load workflow definitions
+shopt -s nullglob
+for workflow in /srv/workflows/*.json; do
+    curl -X POST -H "Content-Type: application/json" http://localhost:8080/api/metadata/workflow -d @$workflow
```
Comment (mchalski) [inline conductor/server/entrypoint/start_conductor.sh]: please note 2 things:
* if a given workflow already exists, 409 conflict will be returned - do we want to handle this?
* workflows a re uploaded in 2 step, first task definitions, then workflow definitions, see e.g.:
https://tracker.mender.io/browse/MEN-1097?focusedCommentId=78777&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-78777

File: conductor/server/entrypoint/start_conductor.sh (hunk 1 above)
Comment (kjaskiewiczz) [inline conductor/server/entrypoint/start_conductor.sh]: done

File: conductor/server/entrypoint/start_conductor.sh (hunk 2 above)
Comment (kjaskiewiczz) [inline conductor/server/entrypoint/start_conductor.sh]: task definitions loading added;
I'm not sure if we want to handle 409

**Pull request merged**